pkg/channel/fanout/fanout_handler_test.go

```diff
@@ -17,15 +17,15 @@ limitations under the License.
 package fanout
 
 import (
+	"context"
 	"errors"
-	"io"
-	"io/ioutil"
 	"net/http"
 	"net/http/httptest"
-	"strings"
 	"testing"
 	"time"
 
+	"github.com/cloudevents/sdk-go"
```

Comment:
@nachocano here too, adding the alias ?

Reply:
Done, thanks!!!

my IDE... everytime I save or run goimports it removes it... adding it back again...